regen/warnings.pl

```diff
@@ -962,6 +973,41 @@ =head2 Default Warnings and Optional Warnings
 Note that neither the B<-w> flag or the C<$^W> can be used to
 disable/enable default warnings.  They are still mandatory in this case.
 
+=head2 "Negative warnings"
+
+As a convenience, you can as of Perl 5.34 pass arguments to the
+C<import()> method both positively and negatively. Negative warnings
+are those with a C<-> sign prepended to their names, positive warnings
```

Comment:
Grammatical nit: Use a semicolon here rather than a comma.

(Same applies in warnings.pm.)

Reply:
Why? How would an extra pixel clarify the meaning?

Reply:
I wouldn’t say it clarifies meaning; it’s just convention in written English that if you have 2 independent clauses—as is the case here—then use either a comma plus conjunction, or a semicolon.

So, either:
> … prepended to their names, and positive warnings …

… or:
> … prepended to their names; positive warnings …